### ISSUE TITLE
Scroll sheet via touch

### ISSUE THREAD

**Issue by seancolsen** [mathesar-foundation/mathesar#2363]
## Current behavior

- On the table page and exploration editor/viewer, the sheet can be scrolled via the mouse.

- On touch-only devices there is no way to scroll

## Desired behavior

- The sheet can be scrolled via touch

- Scrolling via mouse works as it currently does.

**Comment by ratika-12** [comment]
Hey @seancolsen  I want to work on this issue please assign me :-)

**Comment by kgodey** [comment]
Thanks @ratika-12

**Comment by Hatilar420** [comment]
Hi @seancolsen @kgodey , Is this issue already fixed ? As i am able to scroll the table page and exploration page using touch.

**Comment by seancolsen** [comment]
@Hatilar420 No, this issue is not fixed. I'm still able to reproduce it. I'm reproducing using Firefox and Chrome on Linux with Wayland, and also on Android with Firefox and Chrome.

I'm curious to hear what browser and platform you're using.

**Comment by seancolsen** [comment]
Unassigned due to 1 week of inactivity, per our [Contributor Guide](https://wiki.mathesar.org/en/community/contributing). Anyone is welcome to submit a PR for this ticket, even without being assigned.

**Comment by DanyaalMahmood** [comment]
@seancolsen can I work on this?

**Comment by seancolsen** [comment]
Sure @DanyaalMahmood, go ahead.

**Comment by MayaSatishRao** [comment]
@seancolsen @Anish9901 how to reproduce the error? I used the demo table page on both desktop and my mobile and scrolling can be done in both. I am attaching the images of the scroll bars.
In desktop:
![Screenshot 2024-01-02 135152](https://github.com/mathesar-foundation/mathesar/assets/67920373/4bb9b43d-8d30-4b7d-b438-86e271a30498)
In mobile:
![WhatsApp Image 2024-01-02 at 13 51 28](https://github.com/mathesar-foundation/mathesar/assets/67920373/d3b0d7da-9b90-4ed9-a70e-ad48104c4832)

**Comment by seancolsen** [comment]
Wow, interesting. Thanks for this helpful comment, @MayaSatishRao.

After looking into this more closely, I've decided to close this issue.

I _am_ able to scroll via touch on my Android phone within Firefox and Chrome. I don't think I had tried that previously.

I'm still _not_ able to scroll via touch on my touchscreen laptop (Ubuntu, KDE, Wayland, Firefox/Chromium) whereas I _can_ scroll other UI areas via touch (e.g. table inspector). This is strange but seems to be low enough priority that we probably don't need an open issue to track it. Hopefully anyone on a touch-_only_ device will be able to scroll.
